client/src/app/components/audit-template/audit-template-view/audit-template-view.component.scss

```diff
@@ -68,11 +66,38 @@ input[type="time"]::-webkit-calendar-picker-indicator {
   text-align: center;
 }
 
-div.column.no-border > mat-card-title {
-  text-align: center !important;
-}
-
 mat-label {
   color: map-get($alta-primary, A800);
   font-size: 115%;
 }
+
+.position-right {
+  float: right;
+}
+
+::ng-deep .mat-checkbox-checked.mat-accent .mat-checkbox-ripple .mat-ripple-element {
+  opacity: 0.03 !important;
+  background-color: map-get($alta-primary, A100);
+}
+
+::ng-deep .mat-checkbox-checked.mat-accent .mat-checkbox-background, .mat-checkbox-indeterminate.mat-accent .mat-checkbox-background {
+  background-color: map-get($alta-primary, A100);
+}
+
+.button-padding {
+  margin: 5px;
+}
+
+::ng-deep .mat-checkbox-label {
+  padding-top: 5px;
+}
```

Comment:
Can we not use ng-deep here.
Last time we tried to use this, it ended up modifying the scss for the entire material library and it ended up messing up other components

Reply:
I also tend to avoid it but unfortunately for checkbox specifically, I have no choice but to use ng-deep in order to force the style down to the child as it won't overwrite the CSS.